src/views/relative-message.html

```diff
@@ -1 +1,40 @@
-relative-message
\ No newline at end of file
+{% from "components/button.njk" import redcrossButton %}
+{% from "components/progression.njk" import redcrossProgression %}
```

Comment:
This line should go.

Reply:
I suspect it wants pulling out of the one we did earlier too.

Reply:
done